test/example.js

```diff
@@ -138,7 +138,8 @@ const undocumentedDefaultFn = (arg1, arg2 = 'hello, world') => {};
 // Links can have whitespace characters. And you still can use braces inside
 // backticks to, for example, specify object `{ a: 1, b: 2 }`.
 //   link <string> links can be also used across all comments and links names
-//       can have parentheses at the end like `{introspect()}`.
+//       can have parentheses at the end like `{introspect()}` or
+//       `{ExampleClass#method1()}`.
```

Comment:
Maybe also include an example for ``` `{ExampleClass#method1}` ```?

Reply:
`{ExampleClass#method1}` and `{ExampleClass#method1()}` are now equivalent, PTAL

Reply:
I like the idea though I think there should be a way to disable this(making types with and without parentheses equal) through config/cli. IMO it will help to enforce a consistent style. E.g. not having `{ExampleClass#method1}` and `{ExampleClass#method2()}` in different parts of the code. It can be enabled by default, but I feel it will help with making documentation consistent should we need to enforce it in the future.

Reply:
Giving it a second thought this is not needed as it can be enforced by the creator of "customRefs" field. Commit removed.